tests/e2e/Pages/ChatComponent.ts

```diff
@@ -80,12 +80,41 @@ class ChatComponent extends DashboardPage {
     const noTriggerChatDialogue = this.chatDialogue.filter({
       hasText: chatDialogueContentNoAlerts,
     });
-
     // Assertions
     await expect(welcomeChatDialogue).toBeVisible();
     await expect(noTriggerChatDialogue).toBeVisible();
   }
 
+  async chatColumnIsVisibleForTriggerState({
+    firstName,
+    lastName,
+  }: {
+    firstName: string;
+    lastName: string;
+  }) {
+    // String cleaning to remove <strong> tags and replace placeholders with actual values
+    const cleanedString = chatDialogueWarnLabel.replace(/<\/?strong>/g, '');
+
+    const date = new Date();
+    const formattedDate = format(date, 'EEEE, dd MMMM');
+    const formattedTime = format(date, 'HH:mm');
+
+    const lastModelRunDate = `${formattedDate} ${formattedTime}`;
+
+    // Formatted Strings
+    const chatDialogueContent = cleanedString
+      .replace('{{ name }}', `${firstName} ${lastName}`)
+      .replace('{{lastModelRunDate}}', lastModelRunDate);
+
+    // Locators based on data-testid and filtered by formatted strings
+    const welcomeChatDialogue = this.chatDialogue.filter({
+      hasText: chatDialogueContent,
+    });
+
+    // Assertions
+    await expect(welcomeChatDialogue).toBeVisible();
+  }
+
   async allChatButtonsArePresent() {
```

Comment:
I think this method name is no longer correct now right, given that we are now talking about showPrediction button also in the chat component.

Reply:
Changed to: allButtonsArePresent